chrome/content/preferences.xul

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="chrome://global/skin/" type="text/css"?>
+
+<prefwindow id="universalsearch-prefs" title="Universal Search Preferences" xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
+  <prefpane id="unviersalsearch-pane">
+    <preferences>
+      <preference id="pref_server" name="extensions.universalsearch.server" type="string"/>
+    </preferences>
+    <hbox align="center">
+      <label control="server" value="Server: "/>
+      <textbox preference="pref_server" id="server"/>
+    </hbox>
+  </prefpane>
+</prefwindow>
```

Comment:
I found some documentation somewhere that said inline options should only be used when you want prefs to be modifiable by touching pieces of chrome that your addon adds—e.g. toolbar buttons. I can't find that though, so I'll push forward.